### PR TITLE
doc: Update Files section of READMEs from p to s (#660) 

### PR DESCRIPTION
**Pull request by RubenRam**
These are the modified list from the issue checklist:
- [X] pandoc/README.md
- [X] pathman/README.md
- [X] postgres/README.md
- [X] rustlang/README.md
- [X] sass/README.md
- [X] sclient/README.md
- [X] sd/README.md
- [X] setcap-netbind/README.md
- [X] serviceman/README.md
- [X] ssh-adduser/README.md
- [X] ssh-authorize/README.md
- [X] ssh-pubkey/README.md
- [X] ssh-setpass/README.md
- [X] ssh-utils/README.md
- [X] sshd-prohibit-password/README.md
- [X] sttr/README.md
- [X] syncthing/README.md

### CODE REVIEW

File: pathman/README.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ tagline: |
...
 
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/bin/pathman
+```
```
Comment (coolaj86) [inline pathman/README.md:22]: Note to self: I'll also add these:

```text
# Conditionally sources `~/.config/env/PATH.env` in each of
~/.profile
~/.bashrc
~/.zshrc
~/.config/fish/config.fish
```

File: serviceman/README.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ tagline: |
...
 To update or switch versions, run `webi serviceman@stable`
 
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/bin/serviceman
```
Comment (coolaj86) [inline serviceman/README.md:17]: Note to self: I'll make sure these are also mentioned somewhere in the page:

```text
/etc/systemd/system/<PROJECT>.service
~/.config/systemd/user/<PROJECT>.service
~/Library/LaunchAgents/<PROJECT>.plist
/Library/LaunchDaemons/<PROJECT>.plist
HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\Run
```

File: ssh-setpass/README.md [hunk 3]
```diff
@@ -6,6 +6,17 @@ tagline: |
...
 
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/bin/ssh-setpass
+/etc/shadow
```
Comment (coolaj86) [inline ssh-setpass/README.md:17]: This is incorrect. It does not affect `/etc/shadow`.

It affects `~/.ssh/id_rsa`.

Comment (RubenRam) [reply ssh-setpass/README.md]: You're right, fixed.

**Comment by RubenRam** [comment]
@coolaj86 Note pandoc and postgres were missing from previous commit. I've added a clearer PR message this time.

**Review by coolaj86 — APPROVED** [review]
Thanks again!

- changed one path to include the trailing `/`
- switched from `YOUR_USER` to `USERNAME`
- updated the two files I had marked for my own updates

**Comment by coolaj86** [comment]
Deployed to prod 🚀

**Comment by RubenRam** [comment]
@coolaj86 would you mind labeling my last two PR with hacktoberfest and hacktoberfest-accepted? It's a silly thing but, as they were created within the month of October, I could still get the badge. Thanks! :)

**Comment by coolaj86** [comment]
@RubenRam Yes. I meant to do that earlier. Doing it now.

**Pull request merged**